### PR TITLE
Add Shield Acceptance

### LINKED ISSUE #273: Add metric for experiment loading fail/success
This should be a `ni` metric.

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

File: addon/background.js [hunk 1]
```diff
@@ -308,6 +308,39 @@ async function turnOffPrivateWarning() {
   await browser.storage.sync.set({hasSeenPrivateWarning});
 }
 
+let hasSentWidthEvent = false;
+
+async function increaseSidebarMaxWidth() {
+  if (browser.sideview !== undefined) {
```
Comment (pdehaan) [inline addon/background.js]: You’re already doing this check above on L163 and below on L361, so wouldn’t this always be truthy here, in which case is the `else..if` ever reachable?

Comment (johngruen) [reply addon/background.js]: First of all, why are you awake? Second of all...this is meant to just be a PR for the acceptance criteria, but i borked it and brought in this stuff as well.

**Comment by johngruen** [comment]
@pdehaan fixed!

File: docs/acceptance.md [hunk 2]
```diff
@@ -1,41 +1,28 @@
...
-- [ ] If !recently SVed tabs, only show the current tab section.
-- [ ] It should show a feedback button with the query params `ver={add-on-version}`, `ref=panel` and `rel={firefox-release}`.
-- [ ] The browser action icon should accept a fill value of `context-fill`
-- [ ] It should let the user toggle between mobile and desktop UI.
-- [ ] It should remember whether a site has been toggled to/from desktop before and render accordingly.
+## Onboarding
+- [ ] It should show onboarding to outline the basic usage of the feature (outline copy can be borrowed from [here](https://testpilot.firefox.com/experiments/side-view))
+- [ ] Users in `elective_onboarding` branch should only see onboarding on their first interaction with Side View
+- [ ] Users in the `default_onboarding` branch should see onboarding when the Side View shield study is installed
+- [ ] For users in the `elective_onboarding` branch, Side View should *only* be accessible from the browserAction until they are onboarded.
```
Comment (ianb) [inline docs/acceptance.md]: We talked about intercepting first use for elective_onboarding, and then showing the onboarding in the sidebar, with the intended link in a query string or fragment. Then we'll direct the person to their intended destination at the end of onboarding (or if they just try again the second time will skip onboarding)

File: docs/acceptance.md [hunk 3]
```diff
@@ -1,41 +1,28 @@
...
 
-### Private Browsing Considerations
-- [ ] Recent tabs should not be added or updated in Private Browsing Mode.
-- [ ] When a user leaves Private Browsing Mode, the contents of the Panel should be flushed.
+## Survey
+- [ ] It should include the following query params:
+  - [ ] cohort= 1 - 5 depending on segmentation defined above
```
Comment (ianb) [inline docs/acceptance.md]: 1=control, 2=elective, no use, 3=elective, used, 4=default, no use, 5=default, used

**Pull request merged**